hedera-mirror-rest/controllers/contractController.js

```diff
@@ -1074,17 +1093,34 @@ class ContractController extends BaseController {
     transaction,
     contractLogs,
     contractStateChanges,
-    fileData
+    fileData,
+    shouldMockContractResults
   ) => {
     res.locals[constants.responseDataLabel] = new ContractResultDetailsViewModel(
       contractResult,
       recordFile,
       transaction,
       contractLogs,
       contractStateChanges,
-      fileData
+      fileData,
+      shouldMockContractResults
     );
   };
+
+  getMockedContractResultByTransaction = (transaction) => {
```

Comment:
nit: mocked doesn't really inform the scenario
```suggestion
  getDefaultFailureContractResultByTransaction = (transaction) => {
```